### PR TITLE
Added contributors using Github API

### PR DESCRIPTION
**Pull request by supunTE**
Added contributors list to the web site using Github API.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -964,6 +967,7 @@ <h6 class="title">FOSSASIA Links</h6>
         <script src="js/spectragram.min.js"></script>
         <script src="js/smooth-scroll.min.js"></script>
         <script src="js/parallax.js"></script>
-        <script src="js/scripts.js"></script>
+		<script src="js/scripts.js"></script>
```
Comment (abishekvashok) [inline index.html]: Please properly intend the lines.

File: js/contributors.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+var $root = $('html, body');
+$('a').click(function() {
```
Comment (abishekvashok) [inline js/contributors.js]: What is this function for? Pls explain.

File: js/contributors.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+      html += "<img src="+contributors.avatar_url+"><div class='contribs'><p>";
+      html += contributors.contributions;
+      
+      if (contributors.contributions === 1) {
+        html += " contribution";
+      }
+      else {
+        html += " contributions";
+      }
+      $("#contribiii").append(html);
```
Comment (abishekvashok) [inline js/contributors.js]: Please properly name IDs.

**Comment by abishekvashok** [comment]
And squash the commits :)

**Comment by abishekvashok** [comment]
Please match the theme, centerly aling the contributors heading and there are 404s for images. I.e, put only 1 image of a contributor in a uniform standard size.

File: index.html [hunk 4]
```diff
@@ -298,7 +302,6 @@ <h4 class="uppercase mb16">Lightbox Grid Gallery</h4>
 							<img class="col-sm-2" alt="Abishek V Ashok" src="img/abishek.jpg">
 							<img class="col-sm-2" alt="Mochammad Rafi Al Rizkan" src="img/rafi.png">
 							<img class="col-sm-2" alt="Mehmet Çağrı Arı" src="img/cagriari.jpg">
-							<img class="col-sm-2" alt="Reza Juliandri" src="img/reza_juliandri.jpg">
 					</div>
```
Comment (CloudyPadmal) [inline index.html]: Hi @STEdirisuriya ! Why have you removed this line? :)

**Comment by CloudyPadmal** [comment]
Nice work @STEdirisuriya ! Can you address the issue related to different image sizes? And the ones Abishek has pointed out? Will it be too confusing for you to squash all the commits? :) If so you can leave squashing for now and just address the other issues!

File: js/contributors.js (hunk 2 above)
Comment (STEdirisuriya) [reply js/contributors.js]: mistake

File: index.html [hunk 5]
```diff
@@ -19,8 +19,8 @@
         <link href="css/theme-red.css" rel="stylesheet" type="text/css" media="all" />
         <link href="css/custom.css" rel="stylesheet" type="text/css" media="all" />
         <link href='http://fonts.googleapis.com/css?family=Lato:300,400%7CRaleway:100,400,300,500,600,700%7COpen+Sans:400,500,600' rel='stylesheet' type='text/css'>
-    
-	</head>
+        <link href='https://fonts.googleapis.com/css?family=Lato:300,400%7CRaleway:100,400,300,500,600,700%7COpen+Sans:400,500,600' rel='stylesheet' type='text/css'>
```
Comment (abishekvashok) [inline index.html]: Please use the existing font.

File: js/contributors.js (hunk 2 above)
Comment (supunTE) [reply js/contributors.js]: mistake, i'll remvoe it.

File: js/contributors.js (hunk 3 above)
Comment (supunTE) [reply js/contributors.js]: Ok sure.

File: js/contributors.js [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+      html += "<img src='icons/github.png'width='20' height='20'/>"
+      html += "    "+contributors.login;
+      html += "<div class='pic'></a>";
+      html += "<img src="+contributors.avatar_url+"><div class='contribs'><p>";
+      html += contributors.contributions;
+      
+      if (contributors.contributions === 1) {
+        html += " contribution";
+      }
+      else {
```
Comment (abishekvashok) [inline js/contributors.js:18]: What is the use of this if?

**Review by abishekvashok — CHANGES_REQUESTED** [review]

**Comment by the-codingninja** [comment]
Please solve the conflicts

File: js/contributors.js (hunk 6 above)
Comment (supunTE) [reply js/contributors.js:18]: For select which needs to add after number of contributions. There are number of contributions under the contributor. If that number is '1' there must be contribution (singular).
ex:- If ABC's number of contributions is 1 there will be like this
       1 contribution (singular)
       If ABC's number of contributions are more than 1 there will be like this
       2 contributions (plural)

**Comment by supunTE** [comment]
I think it's **OK**

**Comment by supunTE** [comment]
Now

**Comment by supunTE** [comment]
Added contributor list. OK

**Comment by isuruAb** [comment]
Nice work @STEdirisuriya  ,could you please give a preview link (demo link)

File: index.html (hunk 4 above)
Comment (supunTE) [reply index.html]: Nothing It isn't important

**Review by isuruAb — COMMENTED** [review]
Nice work @STEdirisuriya  ,could you please give a preview link (demo link)

**Comment by supunTE** [comment]
@isuruAb What's wrong with this?

**Comment by supunTE** [comment]
Why I can't go ahead.

**Review by supunTE — COMMENTED** [review]
So. What are the errors?

**Comment by poush** [comment]
@STEdirisuriya Provide us a demo link where we can see the changes on the website. This can be generated by going to your Github pages section in the repository settings.

**Review by supunTE — COMMENTED** [review]
Ok. It done

**Comment by supunTE** [comment]
see website with contributor section see there - https://stedirisuriya.github.io/gci17.fossasia.org/
my task -Website: Add and Enhance the "Contributors Section" for the FOSSASIA GCI site by using the Github API ( FOSSASIA )
Please review my task on GCI, FOSSASIA

File: index.html [hunk 7]
```diff
@@ -204,25 +203,129 @@ <h2>Mentors for you</h2>
...
+				 <p>
+		                Follow contributors on social, simpally click their profile names...
+		        </p>
+		        </div>
+					</div>
+					<div class="row contributors-wrapper">			
+					  <div class="members contributers" id="Contributors">
+						<h1 class="section-h1">
+					  </div>
+							<style>
```
Comment (poush) [inline index.html]: Good work, Just one thing. Don't add inline styles. Instead, use CSS classes and stylesheets.

File: index.html [hunk 8]
```diff
@@ -921,6 +991,9 @@ <h6 class="title">Quicklinks</h6>
         <script src="js/spectragram.min.js"></script>
         <script src="js/smooth-scroll.min.js"></script>
         <script src="js/parallax.js"></script>
-        <script src="js/scripts.js"></script>
+	<script src="js/scripts.js"></script>
```
Comment (poush) [inline index.html]: Please follow indentation.

File: index.html [hunk 9]
```diff
@@ -891,22 +912,71 @@ <h6 class="title">Latest tweets</h6>
...
 						<div class="col-md-3 col-sm-6">
 							<div class="widget">
-								<h6 class="title">Quicklinks</h6>
+								<h6 class="title">FOSSASIA Links</h6>
 								<hr>
 								<a href="https://fossasia.org/" class="foss-links"><i class="ti-world" aria-hidden="true"></i> Main Site</a><br>
 								<a href="https://blog.fossasia.org" class="foss-links"><i class="ti-rss" aria-hidden="true"></i> Blog</a><br>
-								<a href="https://codein.withgoogle.com/organizations/fossasia/" class="foss-links"><i class="ti-bag" aria-hidden="true"></i> Google Code-In Page </a><br>
-								<a href="https://events.fossasia.org" class="foss-links"><i class="ti-calendar" aria-hidden="true"></i> Events</a><br>
+								<a href="https://codein.withgoogle.com/organizations/fossasia/" class="foss-links"><i class=" ti-bag" aria-hidden="true"></i> Google Code-In Page </a><br>
```
Comment (poush) [inline index.html]: Don't add unusual spaces.

File: index.html [hunk 10]
```diff
@@ -204,25 +203,129 @@ <h2>Mentors for you</h2>
...
 							<img class="col-sm-2" alt="Yash Kumar Verma" src="img/yashkumarverma.png">
 							<img class="col-sm-2" alt="Aleksandr Lymarenko" src="img/a-v-lymarenko.png">
 							<img class="col-sm-2" alt="Naveen Rajan" src="img/naveen.jpg">
-              <img class="col-sm-2" alt="Kuba Michalski" src="img/kuba-michalski.jpg">
+              						<img class="col-sm-2" alt="Kuba Michalski" src="img/kuba-michalski.jpg">
 							<img class="col-sm-2" alt="Abishek V Ashok" src="img/abishek.jpg">
 							<img class="col-sm-2" alt="Mochammad Rafi Al Rizkan" src="img/rafi.png">
 							<img class="col-sm-2" alt="Mehmet Çağrı Arı" src="img/cagriari.jpg">
-							<img class="col-sm-2" alt="Reza Juliandri" src="img/reza_juliandri.jpg">
-							<img class="col-sm-2" src="img/sabaxuxuna.png" alt="Saba Khukhunashvili">
```
Comment (poush) [inline index.html]: Looks like some images are removed!

File: index.html [hunk 11]
```diff
@@ -204,25 +203,129 @@ <h2>Mentors for you</h2>
...
+		            </div>
+		            
+		        </div>
+		        
+		    </section><section class="bg-secondary">
 		        <div class="container">
     <div class="row">
     <div class="row mb64 mb-xs-24">
 					<div class="col-sm-12 text-center spread-children-large">
-							<img class="col-sm-2" alt="Jin Zi Long" src="img/zilong.jpg">
```
Comment (poush) [inline index.html]: This one is also removed.

**Review by poush — CHANGES_REQUESTED** [review]
Good work. Just some simple things to fix. Please work on requested changes.

File: index.html (hunk 9 above)
Comment (STEdirisuriya) [reply index.html]: OK, sure. Never again

**Review by supunTE — COMMENTED** [review]
ok

**Review by supunTE — COMMENTED** [review]
Fixed

**Review by supunTE — COMMENTED** [review]
fix

**Comment by codedsun** [comment]
Please resolve the conflicts and the requested changes!

**Review by supunTE — COMMENTED** [review]
Okay

File: index.html (hunk 7 above)
Comment (supunTE) [reply index.html]: ok, I use style sheet

Comment (supunTE) [reply index.html]: contributor.css

File: index.html (hunk 11 above)
Comment (supunTE) [reply index.html]: I added it again

File: index.html (hunk 10 above)
Comment (supunTE) [reply index.html]: Ok I added

File: index.html (hunk 9 above)
Comment (supunTE) [reply index.html]: Ok. Now That's OK

File: index.html (hunk 8 above)
Comment (supunTE) [reply index.html]: Indented

**Comment by supunTE** [comment]
I resolved all requests by poush.
I commented some

**Pull request closed without merging**